net-libs/iojs/iojs-1.7.1.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+MY_PV="v${PV}"
+MY_P="${PN}-${MY_PV}"
+
+inherit flag-o-matic pax-utils python-single-r1 toolchain-funcs
+
+DESCRIPTION="An npm compatible platform originally based on node.js"
+HOMEPAGE="http://iojs.org/"
+SRC_URI="http://iojs.org/dist/${MY_PV}/${MY_P}.tar.xz"
+
+LICENSE="Apache-1.1 Apache-2.0 BSD BSD-2 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~x64-macos"
+IUSE="bundled-libs debug icu +npm snapshot +ssl"
+
+RDEPEND="icu? ( dev-libs/icu )
+	!bundled-libs? (
```

Comment:
No, I gave you feedback you never replied to - as well as suggesting a mitigation path forward on IRC.